LoggingKata.Test/TacoParserTests.cs

```diff
@@ -12,17 +12,39 @@ public void ShouldDoSomething()
         }
 
         [Theory]
-        [InlineData("Example")]
+        [InlineData("0, 0, test")]
         public void ShouldParse(string str)
         {
             // TODO: Complete Should Parse
+
+            // Arrange
+            TacoParser tacoparser = new TacoParser();
+
+            // Act
+            ITrackable result = tacoparser.Parse(str);
+
+            // Assert
+            Assert.NotNull(result);
         }
 
         [Theory]
         [InlineData(null)]
         [InlineData("")]
+        [InlineData("not a number,0.0,Taco Bell Test")]
+        [InlineData("0.0,not a number,Taco Bell Test")]
+        [InlineData("0.0,0.0,0.0, Taco Bell Test")]
+        [InlineData("0.0,0.0")]

```

Comment:
The `ShouldFailParse` tests are not all passing at the moment.  See about changing the `Parse` method so that it handles these test cases and returns `null`
![image](https://user-images.githubusercontent.com/37910370/54065199-db597c80-41e2-11e9-8db6-0ab1a51880ac.png)